spec/handlers/newflow/oauth_callback_spec.rb

```diff
@@ -73,5 +73,18 @@ module Newflow
         end
       end
     end
+
+    context 'when authentication found, but the response from social provider does not include email' do
+      let(:request) {
+        create_newflow_user(email)
+        info = { email: nil, name: Faker::Name.name }
+        MockOmniauthRequest.new 'facebook', Faker::Internet.uuid, info
+      }
+
+      it 'doesn\'t blow up' do
```

Comment:
how about it 'processes the omniauth request without error'?  doesnt blow up doesnt add much information to the it block when you look at the rspec test output.

Reply:
Thanks for the suggestion. Fixed.